webapp/src/js/services/rules-engine.js

```diff
@@ -66,7 +66,7 @@ var nools = require('nools'),
             return contactHasId(fact.contact, factId);
           });
           if (!fact) {
-            fact = new Contact({ reports: [] });
+            fact = new Contact({ contact: { _id: factId }, reports: [] });
```

Comment:
Nice one.

Reply:
It isn't 100% backward compatible. In task code, if partners look for headless reports by testing for `!!c.contact`, it will now be this skeleton definition. But unsure if anybody actually does this outside muso?